appveyor.yml

```diff
@@ -31,23 +32,23 @@ environment:
         CONDA_DEPENDENCIES: "scipy glymur"
         CONDA_CHANNELS: "conda-forge astrofrog"
         PIP_DEPENDENCIES: "requests pyparsing"
-        CONDA_CHANNEL_PRIORITY: "True"
+        CONDA_CHANNEL_PRIORITY: "flexible"
         DEBUG: "True"
 
       - PYTHON_VERSION: "3.6"
         NUMPY_VERSION: "stable"
         ASTROPY_VERSION: "stable"
         CONDA_DEPENDENCIES: "matplotlib h5py"
         PIP_DEPENDENCIES: "astrodendro"
-        CONDA_CHANNEL_PRIORITY: "False"
+        CONDA_CHANNEL_PRIORITY: "disabled"
         SUNPY_VERSION: "dev"
         DEBUG: "True"
 
         # For the Numpy dev build, we also specify a conda package that depends
         # on Numpy to make sure that Numpy dev is used (because conda will also
         # install Numpy stable)
 
-      - PYTHON_VERSION: "3.4"
+      - PYTHON_VERSION: "3.7"
         NUMPY_VERSION: "1.9"
```

Comment:
I'm surprised the combination Python 3.7/numpy 1.9 works!

Reply:
Surprised, but numpy 1.9 is available on python 3.7

Reply:
I would keep as long as it works.